src/op_mode/bridge.py

```diff
@@ -159,6 +161,17 @@ def _get_formatted_output_mdb(data):
     output = tabulate(data_entries, headers)
     return output
 
+def _get_bridge_details(iface):
+    """Get interface detail statistics"""
+    return call(f'vtysh -c "show interface {iface}"')
+
+def _get_bridge_nexthop_group(iface):
+    """Get interface detail nexthop_group statistics"""
+    return call(f'vtysh -c "show interface {iface} nexthop_group"')
```

Comment:
```
root@r14:/home/vyos# ./bridge.py show_bridge --interface=br0 --nexthop_group 
% Unknown command: show interface br0 nexthop_group
1
root@r14:/home/vyos# 
```

`nexthop_group` incroccert parameter for the FRR
expected dash `nexthop-group`

Reply:
done The command is wrong